src/redux/rocketReducer.js

```diff
@@ -0,0 +1,41 @@
+const initialState = {
+  rockets: [
+    {
+      id: 1,
+      name: 'Falcon 1',
+      description: 'Description of Falcon 1',
+      reserved: false,
+    },
+    {
+      id: 2,
+      name: 'Falcon 2',
+      description: 'Description of Falcon 2',
+      reserved: false,
+    },
+  ],
+};
+
```

Comment:
Update your **rocketReducer.js** to handle the **FETCH_ROCKETS** action and store the rocket data in the Redux store. Here is an example: 
```js
// Import the action type
import { FETCH_ROCKETS } from './rocketActions';

const initialState = {
  rockets: [],
};

const rocketReducer = (state = initialState, action) => {
  switch (action.type) {
    case FETCH_ROCKETS:
      return { ...state, rockets: action.payload };

    // ... Other reducer cases ...
  }
};

export default rocketReducer;

```